sgkit/tests/test_aggregation.py

```diff
@@ -289,3 +292,64 @@ def test_sample_stats(precompute_variant_allele_count):
     np.testing.assert_equal(ss["sample_n_hom_alt"], np.array([0, 1]))
     np.testing.assert_equal(ss["sample_n_het"], np.array([3, 1]))
     np.testing.assert_equal(ss["sample_n_non_ref"], np.array([3, 2]))
+
+
+def test_infer_call_ploidy():
+    ds = get_dataset(
+        [
+            [[0, 0, 0, 0], [0, 1, -2, -2], [1, 0, 0, 0]],
+            [[-1, 0, -2, -2], [0, -1, -1, -1], [-1, -1, -1, -1]],
+        ],
+        n_ploidy=4,
+    )
+    # test as fixed ploidy
+    cp = infer_call_ploidy(ds).call_ploidy
+    np.testing.assert_equal(cp, np.array([[4, 4, 4], [4, 4, 4]]))
```

Comment:
Just as a passing thing: I don't think you need to explicitly convert the second array to numpy, 
```python
  np.testing.assert_equal(cp, [[4, 4, 4], [4, 4, 4]])
```
also works.

Reply:
It's actually throwing `ValueError: The truth value of an array ...` when used with `xr.DataArray` and `list`. `cp.values` would work but calling `np.array` is consistent with the other tests.

Reply:
Ah, I didn't try with an xr array. My bad!